Godeps/Godeps.json

```diff
@@ -6,13 +6,26 @@
 	],
 	"Deps": [
 		{
-			"ImportPath": "github.com/coreos/fleet/version",
-			"Comment": "v0.3.2-1055-ga35ee29",
-			"Rev": "a35ee29f4f539d3d2296ac85c302f8b2e7d7a878"
+			"ImportPath": "code.google.com/p/goauth2/oauth",
```

Comment:
So the interface has changed, and the version used here is deprecated :/ going to take a while to figure out the new interface. *Looks at clock* must sleep!